public/app/features/plugins/plugin_loader.ts

```diff
@@ -118,7 +119,18 @@ export async function importPluginModule({
     return importPluginModuleInSandbox({ pluginId });
   }
 
-  return SystemJS.import(modulePath);
+  return SystemJS.import(modulePath).catch((e) => {
+    let error = new Error('Could not load plugin: ' + e);
+    console.error(error);
+    pluginsLogger.logError(error, {
+      path,
+      pluginId,
+      pluginVersion: version ?? '',
+      expectedHash: moduleHash ?? '',
+      sriChecksEnabled: (config.featureToggles.pluginsSriChecks ?? false).toString(),
```

Comment:
Do we want to track `loadingStrategy` here too? 🤔

Reply:
Good point, I've added it! :+1: